pandas/tests/arrays/test_array.py

```diff
@@ -0,0 +1,118 @@
+import decimal
+
+import numpy as np
+import pytest
+
+from pandas.core.dtypes.dtypes import registry
+
+import pandas as pd
+from pandas.api.extensions import register_extension_dtype
+from pandas.core.arrays import integer_array, period_array
```

Comment:
is decimal and json mentioned as first class dtypes? e.g. in the api? should we?

Reply:
Those are just test classes. They aren't part of the API.

Reply:
I think an arrow-backed Decimal array would make sense (this one is backed by an object-dtype ndarray).

Reply:
sure arrow backed things make sense, but until kernels are fully available an object based one is just nice to have. (and even then could just be a backend swap)